widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -496,7 +496,7 @@ public void videoButtonClicked() {
         Logger.d(TAG, "videoButtonClicked");
         if (callState.isCallOngoingAndOperatorConnected()) {
             VisitorMediaState currentMediaState = callState.callStatus.getVisitorMediaState();
-            Logger.d(TAG, "videoButton status:" + currentMediaState.getAudio().getStatus().toString());
+            Logger.d(TAG, "videoButton status:" + currentMediaState.getVideo().getStatus().toString());
```

Comment:
Is it necessary to keep the log code in the source? Ideally you would not have logging in production code.

Reply:
@jakob-mc  it makes sense, but this logging is under `debug` level, so it will be ignored for production code.
I could be wrong, so if I missed anything, please clarify once more your suggestion

Reply:
I was thinking if anybody benefits from the logs after the feature/fix have been implemented and tested? If they are only helpful during the task development, then should be thrown out before merging the finished task.

Reply:
imho, logging is useful for debugging purposes anytime. It should not go to the production with info/debug/trace levels, but I'm not sure that we need to remove logging at all.

Reply:
I'd prefer to go with logging as it has saved us a lot of time previously :)

Reply:
clear, just wanted to make sure you really benefit from them.